coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -1,3 +1,10 @@
+"""
+Language and docstyle independent documentation comments extraction.
+
+Each of the functions is built upon one another, and at the last,
+exposes a single function py:function:`extract_documentation_with_markers`
```

Comment:
> Sorry, I don't quite get what you mean. I didn't add a reference to function? I added py:function:extract_documentation_with_markers

Sorry this got a little function, you dont need the actual `py` prefix, also the shorthand is `func`, so something like 
```
:func:`extract_documentation.......`
``` 
should work. Also class reference:

```
:class:`.DocBaseClass`
```

Notice the dot before the class name. that helps sphinx search the class in the same module :)